packages/b/bchaddrjs.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "bchaddrjs",
+  "description": "Bitcoin Cash general purpose address translation.",
+  "keywords": [
+    "bitcoin-cash",
+    "bitcoin",
+    "bch",
+    "cryptocurrency",
+    "address",
+    "encoding",
+    "translation",
+    "base58",
+    "bitpay",
+    "cash-address"
+  ],
+  "author": {
+    "name": "Emilio Almansi",
+    "email": "hi@ealmansi.com"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/ealmansi/bchaddrjs#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/ealmansi/bchaddrjs.git"
+  },
+  "npmName": "bchaddrjs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.js"
+      ]
+    }
+  ]
+}
```

Comment:
there's no filename. I believe it will only have an impact on the website where we don't show the default?

should we warn?

Reply:
Yeah, filename is only used to highlight a default file. This package has the file names with versions in them, so there isn't anything I can set as the default.

I think warning would be okay, but it shouldn't block the PR passing.